package.json

```diff
@@ -47,12 +47,12 @@
     "@tanstack/react-query-persist-client": "^4.36.1",
     "@tanstack/react-table": "^8.17.3",
     "@thirdweb-dev/chains": "0.1.116",
-    "@thirdweb-dev/react": "4.6.31",
-    "@thirdweb-dev/react-core": "4.6.31",
-    "@thirdweb-dev/sdk": "4.0.92",
+    "@thirdweb-dev/react": "0.0.0-dev-425a6372875caa6cebee887d821887a92f793a08-20240530183645",
+    "@thirdweb-dev/react-core": "0.0.0-dev-425a6372875caa6cebee887d821887a92f793a08-20240530183645",
+    "@thirdweb-dev/sdk": "0.0.0-dev-425a6372875caa6cebee887d821887a92f793a08-20240530183645",
```

Comment:
do we still need dev versions for this? Can we instead just merge these changes into stable SDK first?